Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -182,8 +184,42 @@ enum PackageShow {
                         source: SiteURL.package(.value(model.repositoryOwner),
                                                 .value(model.repositoryName),
                                                 .readme).relativeURL(),
-                        // Until the content is loaded, substitute a spinner.
-                        .spinner()
+                        .group(
+                            .data(named: "tab-page", value: "readme"),
+                            // Until the content is loaded, substitute a spinner.
+                            .spinner()
+                        )
+            )
+        }
+        
+        func releaseSection() -> Node<HTML.BodyContext> {
+            .turboFrame(id: "releases",
+                        source: SiteURL.package(.value(model.repositoryOwner),
+                                                .value(model.repositoryName),
+                                                .releases).relativeURL(),
+                        .group(
+                            .data(named: "tab-page", value: "releases"),
+                            // Until the content is loaded, substitute a spinner.
+                            .spinner()
+                        )
+            )
+        }
+        
+        func tabBar() -> Node<HTML.BodyContext> {
+            .group(
+                .element(named: "tab-bar", nodes: [
```

Comment:
Let's add a `Node` extension that defines `tab-bar`, as we have with `spi-readme`. The `ul` should still be in this file, just the `tab-bar` element.

We also might want to call it `spi-tab-bar`, just in case any extensions or similar define a `tab-bar` element.